src/main/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminRole.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+final class AdminRole {
+    static final String FACT_SUPER_ADMIN = "fact-super-admin";
```

Comment:
One to perhaps discuss and do later on, but do we want to include these in the Dockerfile as environment variables and configure them? 

This'll be fine locally, but is there a process higher up for this? Feel free to resolve if it's already been considered ofc